lib/system-task.js

```diff
@@ -14,7 +14,7 @@ const OP_DATA = 0b001;
 const OP_STAT = 0b010;
 const OP_REAL = 0b100;
 
-module.exports.op = {
+const op = {
```

Comment:
Why was this changed?

Reply:
Fixed in https://github.com/file-icons/fs/pull/8/commits/7b6329b9ffeee2702883b9bee92322ad101a676d